### PR TITLE
Update rubygem-pulp_2to3_migration_client to 0.5.0

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Review by chris1984 — APPROVED** [review]

**Comment by chris1984** [comment]
@tbrisker this is needed for 3.17

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 1]
```diff
@@ -21,13 +21,14 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 1.9
-Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
```
Comment (tbrisker) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec:26]: This seems redundant with the next line, was the specfile generated by obal or manually? (i.e. is this a mistake in the gemspec?)

Comment (jlsherrill) [reply packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec:26]: it was auto-generated using 'bump_rpm.sh'.  Looks like this is a quirk of the spec file: https://rubygems.org/gems/pulp_2to3_migration_client

Comment (tbrisker) [reply packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec:26]: huh, odd. I tried to find where the gemspec is but seems like it's generated in some automated process as part of the version release. i couldn't find where it got these requirements from.

**Review by tbrisker — APPROVED** [review]
spec is odd but looks like it's true to the gemspec.

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec (hunk 1 above)
Comment (evgeni) [reply packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec:26]: Oh yeah, if the gem has both >= and ~> in the spec, this looks like what you'd get.

Not nice, but also not hurting.

**Review by evgeni — APPROVED** [review]
Will need the koji 💃, so not merging right now.

**Pull request merged**